src/core/src/pass/sdpa_to_paged_attention.cpp

```diff
@@ -42,15 +44,23 @@ bool ov::pass::SDPAToPagedAttention::run_on_model(const std::shared_ptr<ov::Mode
                     "No ScaledDotProductAttention operation observed in the graph, cannot perform "
                     "the SDPAToPagedAttention transformation.");
 
+    std::map<std::string, std::shared_ptr<v0::Parameter>> optional_model_wide_params;
+
     auto max_context_len = setName(std::make_shared<v0::Parameter>(element::i32, PartialShape{}), "max_context_len");
-    ParameterVector model_remaining_params{
+    ParameterVector model_wide_params{
         setName(std::make_shared<v0::Parameter>(element::i32, PartialShape{-1}), "past_lens"),
         setName(std::make_shared<v0::Parameter>(element::i32, PartialShape{-1}), "subsequence_begins"),
         setName(std::make_shared<v0::Parameter>(element::i32, PartialShape{-1}), "block_indices_begins"),
     };
     if (!m_use_per_layer_block_indices_inputs) {
         auto block_indices = setName(std::make_shared<v0::Parameter>(element::i32, PartialShape{-1}), "block_indices");
-        model_remaining_params.insert(model_remaining_params.begin() + 2, block_indices);
+        model_wide_params.insert(model_wide_params.begin() + 2, block_indices);
+    }
+
+    if (m_allow_score_aggregation) {
+        auto score_aggregation_window =
+            setName(std::make_shared<v0::Parameter>(element::i32, PartialShape{-1}), "score_aggregation_window");
+        optional_model_wide_params["score_aggregation_window"] = score_aggregation_window;
```

Comment:
do we have a tests when 

bool allow_score_aggregation = false
bool allow_cache_rotation = true

and 
bool allow_score_aggregation = true
bool allow_cache_rotation = false

and 
bool allow_score_aggregation = true
bool allow_cache_rotation = true

Reply:
Which tests do you have in mind?